api/db/populate-events.js

```diff
@@ -71,15 +70,15 @@ const createEvents = async (multiplier, randomLocation, verbose) => {
   const now = new Date();
   const futureDate = new Date().setFullYear(now.getFullYear() + 5);
 
-  for (let i = 0; i < 500 * multiplier; i++) {
+  for (let i = 0; i < 50 * multiplier; i++) {
```

Comment:
Please note that this change (and the users' one) requires a small update on the API's README.